README.rst

```diff
@@ -188,6 +194,24 @@ if all migrations have already been run::
     $ dbmigrator migrate
     No pending migrations.  Database is up to date.
 
+To write a repeat migration, make sure your migration has ``should_run`` defined::
+
+    def should_run(cursor):
+        return os.path.exists('data.txt')
+
+
+    def up(cursor):
+        with open('data.txt') as f:
+            data = f.read()
+        cursor.execute('INSERT INTO table VALUES (%s)', (data,))
+
+
+    def down(cursor):
```

Comment:
That's what "should_run" is, you can add a check to "up" for non repeatable migrations?